src/taipy/core/config/_service_config.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2023 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+from copy import copy
+from typing import Any, Dict, List, Optional, TypedDict
+
+from taipy.config import Config, UniqueSection
+from taipy.logger._taipy_logger import _TaipyLogger
+
+from .._version._cli._core_cli import _CoreCLI
+
+ServiceConfig = TypedDict(
+    "ServiceConfig",
+    {
+        "mode": str,
+        "version_number": str,
+        "force": bool,
+        "clean_entities": bool,
+    },
+    total=False,
+)
+
+
+default_service_config: ServiceConfig = {
+    "mode": "development",
+    "version_number": "",
+    "force": False,
+    "clean_entities": False,
+}
+
+
+class _ServiceConfig:
```

Comment:
I don't get the purpose of this class _ServiceConfig while you already have the CoreSection that can do the job.

Reply:
This is to retain the behavior as the GUI section:
- The `CoreSection` represents a unique section the Config.
- The `_ServiceConfig` contains information needed to start the service, which is only a small dictionary in Core (but we can have more in the future).

I don't think we should merge the two classes.

Reply:
Indeed, it is a section in the COnfig, but for me, the CoreSection represents the information needed to start the Core service as well. No?

Reply:
You are right. In GUI, the GUI service needs several dictionaries of "configs", not just the GUISection, but we only need the CoreSection.

I will try to merge these 2 classes together.